tests/big_int.rs

```diff
@@ -0,0 +1,76 @@
+#![cfg(test)]
+
```

Comment:
this file is already in `tests` folder, so Cargo will process this folder only in test cfg mode
```suggestion
```